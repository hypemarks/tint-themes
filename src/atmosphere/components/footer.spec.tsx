import React from 'react';
import { render } from '@testing-library/react';
import Footer from './footer';

jest.mock('./popup-circle', () => ({ LinkType: jest.fn() }));

describe('Test Footer Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <svg>
        <Footer font='lato' fontColor='red' position={{ x: 23, y: 23 }} time='' formatter={''} width={23} />
      </svg>
    );
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
