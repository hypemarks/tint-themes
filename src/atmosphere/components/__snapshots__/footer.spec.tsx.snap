// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer Component should render snapshot 1`] = `
<DocumentFragment>
  <svg>
    <text
      align="center"
      fill="red"
      font-family="lato"
      text="NaN year from now"
      width="23"
      x="23"
      y="23"
    />
  </svg>
</DocumentFragment>
`;
